remotespark/datawidgets/encoding.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+
+class Encoding(object):
```

Comment:
It's a name I learned while working with Altair. We'll keep as is to align to Altair as much as possible.